supabase/migrations/20230404202715_add_project_columns.sql

```diff
@@ -0,0 +1,21 @@
+alter table public.projects 
+alter COLUMN "id" type text,
+add COLUMN "handle" text,
+add COLUMN "projectId" int,
+add COLUMN "pv" char(1) check (pv in('1', '2')),
+add COLUMN "currentBalance" char(32),
```

Comment:
**issue** i think we should snake case all these names to be consistent: https://discord.com/channels/939317843059679252/939705688563810304/1092316422136418355

Reply:
ahhh forgot about this. can do